src/main/resources/public/index.html

```diff
@@ -40,7 +40,7 @@ <h1 class="cover-heading">Food Agency Data Summarizer.</h1>
 
     <footer class="mastfoot mt-auto">
         <div class="inner">
-            <p><a href="http://www.inifinityworks.com">Infinity Works</a> - Full Stack Tech Test. <a
+            <p><a href="http://www.infinityworks.com">Infinity Works</a> - Full Stack Tech Test. <a
```

Comment:
suggestion: The other article linked here also no longer exists so gets redirected to the general insights page. Should we just link to https://www.infinityworks.com/careers/ instead?

Reply:
Sounds good, done.